main/main.cpp

```diff
@@ -621,6 +621,9 @@ void Main::print_help(const char *p_binary) {
 	print_help_title("Debug options");
 	print_help_option("-d, --debug", "Debug (local stdout debugger).\n");
 	print_help_option("-b, --breakpoints", "Breakpoint list as source::line comma-separated pairs, no spaces (use %%20 instead).\n");
+#ifdef DEBUG_ENABLED
+	print_help_option("--fail-on-error", "Exit with a failure code if any errors are encountered.\n", CLI_OPTION_AVAILABILITY_TEMPLATE_DEBUG);
+#endif
```

Comment:
This could be exposed in release builds too, since the other code added in this PR isn't restricted to debug builds.

Reply:
Sure. I'm fine having it in all builds.